MobileFilter/sections/specific_app.txt

```diff
@@ -330,6 +331,8 @@
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/126099
+||adsgard.aprod.scopely.io^$app=com.pieyel.scrabble
```

Comment:
What about another games of this author?

Reply:
And missed trecker:
https://489a19fa-d831-4954-858d-46436d5cdb20.event.collector.scopely.io